tests/unittest_qmail-send.c

```diff
@@ -0,0 +1,239 @@
+#define DEPRECATED_FUNCTIONS_REMOVED
+#include <check.h>
+#include "qmail-send-without-main.c"
+
+extern struct job *jo;
+extern int numjobs;
+
+// START test helpers for the API we might prefer from qmail-send
+
+#define available 0
+#define unavailable 1
+
+static void job_init_this_many(int numslots) {
+  numjobs = numslots;
+  job_init();
+}
+
+static void occupy_job(unsigned int index) {
+  jo[index].refs = unavailable;
+}
+
+static void occupy_all_jobs(void) {
+  for (int j = 0; j < numjobs; j++) {
```

Comment:
You need to declare the variable on it's own as it's otherwise not valid C89 code. We have not yet raised the minimum required version to C99.

Reply:
This is good to know, thanks for this insight. I thought we were using C99.